demos/projects/ESPRESSIF/esp32/components/sample-azure-iot/transport_tls_esp32.c

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file tls_freertos.c
+ * @brief TLS transport interface implementations. This implementation uses
+ * mbedTLS.
+ */
+
+/* Standard includes. */
+#include <string.h>
+
+/* FreeRTOS includes. */
+#include "freertos/FreeRTOS.h"
+
+/* TLS transport header. */
+#include "transport_tls_socket.h"
+
+#include "esp_log.h"
+
+/* TLS includes. */
+#include "esp_transport_ssl.h"
+
+/**
+ * @brief Definition of the network context for the transport interface
+ * implementation that uses mbedTLS and FreeRTOS+TLS sockets.
+ */
+struct NetworkContext
+{
+    esp_transport_handle_t transport;
+    uint32_t receiveTimeoutMs;
+    uint32_t sendTimeoutMs;
+};
+
+static const char *TAG = "tls_freertos";
+/*-----------------------------------------------------------*/
+
+TlsTransportStatus_t TLS_Socket_Connect( NetworkContext_t * pNetworkContext,
+                                           const char * pHostName,
+                                           uint16_t port,
+                                           const NetworkCredentials_t * pNetworkCredentials,
+                                           uint32_t receiveTimeoutMs,
```

Comment:
indentation.

plus can you please use following guidline, when naming things:

[https://www.freertos.org/FreeRTOS-Coding-Standard-and-Style-Guide.html](https://www.freertos.org/FreeRTOS-Coding-Standard-and-Style-Guide.html)

Reply:
All fixed. Including variable names.